openstates/me/bills.py

```diff
@@ -194,21 +194,31 @@ def scrape_bill(self, bill):
         if srow:
             bill['subjects'] = [s.strip() for s in srow if s.strip()]
 
+        # Attempt to find link to bill text/documents.
         ver_link = page.xpath("//a[contains(@href, 'display_ps.asp')]")[0]
         ver_url = ver_link.get('href')
+
         try:
             ver_html = self.get(ver_url, retry_on_404=True).text
-        except socket.timeout:
+        except (socket.timeout, requests.exceptions.HTTPError):
             pass
         else:
             if ver_html:
                 vdoc = lxml.html.fromstring(ver_html)
-                vdoc.make_links_absolute(ver_url)
-                # various versions: billtexts, billdocs, billpdfs
-                vurl = vdoc.xpath('//a[contains(@href, "billtexts/")]/@href')
-                if vurl:
-                    bill.add_version('Initial Version', vurl[0],
-                                     mimetype='text/html')
+
+                # Check whether the bill text is missing.
+                bill_text_missing = vdoc.xpath('string(//div[@id = "sec0"])').strip()
+                match = re.search('Cannot find requested paper', bill_text_missing)
+
+                if not match:
+                    vdoc.make_links_absolute(ver_url)
+
+                    # various versions: billtexts, billdocs, billpdfs
+                    vurl = vdoc.xpath('//a[contains(@href, "billtexts/")]/@href')
+
+                    if vurl:
+                        bill.add_version('Initial Version', vurl[0],
```

Comment:
I made this change on a separate branch, which I'll make a pull request for in just a moment.